packages/components/src/widgets/DistWidget/DistributionsChart.tsx

```diff
@@ -497,6 +499,11 @@ export const DistributionsChart: FC<DistributionsChartProps> = ({
     isNormalized: distribution.isNormalized(),
   }));
 
+  const showAxisTitles = height > 30;
+  const shownXAxisTitle = showAxisTitles && !!plot.xScale.title;
+  const showXAxis = height > 15;
+  const nonTitleHeight = Math.max(height - (showXAxis ? 20 : 0), 0);
```

Comment:
This line looks sketchy to me, not sure where `20` is coming from, probably the dependency on the internals of `InnerDistributionsChart`?